tools/accuracy_checker/configs/person-detection-asl-0001.yml

```diff
@@ -0,0 +1,43 @@
+models:
+  - name: person-detection-asl-0001
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/person-detection-asl-0001/FP32/person-detection-asl-0001.xml
+        weights: intel/person-detection-asl-0001/FP32/person-detection-asl-0001.bin
+        adapter:
+          type: fcos_person
+          output_blob: 12688/Split.0
+          scale: 0.003125
+
```

Comment:
please add FP16 launcher

Reply:
Unfortunately we have FP32 IR for the model only... In future it will be replaced on proper variant but for now it is it.